src/dvui.zig

```diff
@@ -2437,7 +2441,10 @@ pub const Window = struct {
             self.debug_under_mouse_info = "";
         }
 
+        _ = self._arena.reset(.retain_capacity);
```

Comment:
previously you used `.free_all`. that's not how arena allocator should be used.

I also moved this call to the start of the frame.